spec/models/course/level_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Course::Level, type: :model do
+  it { is_expected.to belong_to(:course).inverse_of(:levels) }
+
+  let(:level) { Course::Level.new }
+  it 'rejects negative exp_threshold' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.